device-types/Cisco/Cisco SMA M395.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Cisco
+model: SMA M395
+slug: cisco-sma-m395
+part_number: SMA-M395
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: RPC
+    type: rj-45
+  - name: usb1
```

Comment:
Are these console USB ports, or general io/storage ports? the later would not be modelled in Netbox at this time.

Reply:
I believe they are usb to serial ports for people that don't have the serial ports on their computers.  I think you use a special usb cable and software to access the console.